test/unit/content-info.test.js

```diff
@@ -23,6 +23,23 @@ describe('contentInfo module', function () {
         });
     });
 
+    // eslint-disable-next-line max-len
+    it('For mp3 files response, it should take the sniffed content-type from the response stream if content-type and content-disposition headers is not present', function () {
```

Comment:
`Should detect mp3 response stream if content-type is not present`

Reply:
done